packages/kit/src/views/ReferFriends/pages/InviteReward/components/RulesButton.tsx

```
@@ -0,0 +1,24 @@
+  const intl = useIntl();
+
+  const handlePress = () => {
+    void openUrlExternal(REFERRAL_HELP_LINK);
+  };
+
+  return (
+    <Button
+      variant="tertiary"
+      icon="QuestionmarkOutline"
+      onPress={handlePress}
+    >
```

ESLint

[error] Replace `⏎······variant="tertiary"⏎······icon="QuestionmarkOutline"⏎······onPress={handlePress}⏎····` with `·variant="tertiary"·icon="QuestionmarkOutline"·onPress={handlePress}`

(prettier/prettier)